packages/perseus/src/perseus-types.ts

```diff
@@ -645,7 +645,7 @@ export type PerseusInteractiveGraphWidgetOptions = {
     correct: PerseusGraphType;
     // Shapes (points, chords, etc) displayed on the graph that cannot
     // be moved by the user.
-    lockedFigures?: ReadonlyArray<LockedFigure>;
+    lockedFigures?: Array<LockedFigure>;
```

Comment:
Ideally, the types here should remain read-only. These are the public API of Perseus and represent the data passed into Perseus. As such, I think we should treat them as immutable and copy them first if we want to mutate them.

Why did we need to make it mutable?

Reply:
+1!

Reply:
Ah, I changed it so it's more straightforward here:

https://github.com/Khan/perseus/pull/1067#discussion_r1522306486

But I hear you on the public API needing to be immutable. I'll go back to the first approach.